commit ea4ce8e3e7ef30ea93ed4fde8851c58bfaab77d9 (refs/changes/74/4574/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T20:11:37+00:00 (1 year, 2 months ago)
    
    TRITON-612 fw-agent should be aware of alternate admin nictags (bump version)
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 61ef70a..f8a2650 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "firewaller",
     "description": "SmartDataCenter Firewalling Agent",
-    "version": "1.4.0",
+    "version": "1.5.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
